sklearn/linear_model/logistic.py

```diff
@@ -1198,6 +1198,14 @@ def fit(self, X, y, sample_weight=None):
         self : object
             Returns self.
         """
+        if self.solver == 'auto':
+            self.solver = 'liblinear'
+            warnings.warn("Default solver will be changed to 'lbfgs' in 0.22",
+                          FutureWarning)
+        if self.multi_class == 'auto':
+            self.multi_class = 'ovr'
+            warnings.warn("Default multi_class will be changed to "
```

Comment:
Only when solver is not liblinear

Reply:
But when solver is indeed liblinear, multi_class remains as auto and thus raises an error. Am I missing out something here?